### DISCUSSION TITLE
4 Hidden Pictures But Nothing in Browser Interface

### DISCUSSION THREAD

**Discussion by erikvanderwerf** [photoprism/photoprism#4642] [Q&A]
Hello,

My bowser interface indicates 4 Hidden photos, but I cannot find them.

![image](https://github.com/user-attachments/assets/5485d3ca-f768-4b73-b8e3-b17378bb5ede)

What is the recommended action here?

**Comment by omerdduran** [comment]
Hi. Files can be marked as "hidden" when PhotoPrism can't properly process them. Specifically, if a file has an incorrect extension (like an HEIC file with .jpg extension), it may be counted as hidden but won't appear in the UI because PhotoPrism couldn't create the necessary thumbnail/sidecar files. You can look at the logs and find out which photos they are.

**Reply by erikvanderwerf** [reply]
I've looked in the log and found a few that look promising, except that I have 7 logs similar to the below, which does not match the number of 4 Hidden photos, so I am not sure this is correct.

![image](https://github.com/user-attachments/assets/79b1c7ed-2f6f-4395-8e7d-1fb4d35d7076)

> import: $file has invalid extension for media type 'image/webp'

Even so, my imports folder no longer contains $file (if it existed in the first place). I believe PhotoPrism has already moved this file and renamed it, so I'm not sure what to do now.
